components/script/dom/webidls/HTMLImageElement.webidl

```diff
@@ -9,8 +9,8 @@ interface HTMLImageElement : HTMLElement {
            attribute DOMString alt;
   [CEReactions]
            attribute DOMString src;
-  // [CEReactions]
-  //         attribute DOMString srcset;
+  [CEReactions]
```

Comment:
In order to generate appropriate bindings for the return type, you'll need to add the `[Throws]` attribute here.